include/eve/arch/wasm/spec.hpp

```diff
@@ -0,0 +1,37 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/arch/wasm/predef.hpp>
+
+#include <cstddef>
+#include <cstdint>
+
+//==================================================================================================
+// WASM Info
+#if defined(EVE_HW_WASM)
+namespace eve
+{
+struct register_count
+{
+  static constexpr std::size_t general = 2 * sizeof(void *);
+  static constexpr std::size_t simd    = 2 * sizeof(void *);
+};
```

Comment:
can you remind me what this is? especially * sizeof(void *)? are you being too cute with no comments?